app/Http/Controllers/Api/SignupsController.php

```diff
@@ -50,7 +50,12 @@ public function store(SignupRequest $request)
     {
         $transactionId = incrementTransactionId($request);
 
-        $signup = $this->signups->create($request->all(), $transactionId);
+        // Check to see if the signup exists before creating one.
+        $signup = $this->signups->get($request['northstar_id'], $request['campaign_id'], $request['campaign_run_id']);
+
+        if (! $signup) {
+            $signup = $this->signups->create($request->all(), $transactionId);
+        }
 
         if ($signup) {
             $code = 201;
```

Comment:
Let's send a `200 OK` when a signup already exists, and only send a `201 Created` if creating a new one!